scripts/dev_cli.sh

```diff
@@ -4,8 +4,9 @@ IFS=$'\n\t'
 
 brew install -q coreutils wget
 
+VERSION=$(cat ./VERSION)
 AIRY_BIN=/usr/local/bin/airy_dev
-BUCKET=https://airy-core-binaries.s3.amazonaws.com/develop/darwin/amd64
+BUCKET=https://airy-core-binaries.s3.amazonaws.com/${VERSION}/darwin/amd64
```

Comment:
for that to work, I think you also need to change the upload-cli-binary.sh to upload an alpha cli binary on merge to develop

Reply:
Correct. I think this is it
https://github.com/airyhq/airy/pull/2241/files#diff-10eb59b0cdafa7a3fe250423fffb995b31a284266a6b04191eb090e92f9b4276

Reply:
this only changes which Version is backed into the binary but the file itself is not created by the CI I think

Reply:
You are rignt, this is for the bucket where the binary is actually uploaded to and it differs from that is set in the `upload-cli-binary.sh` script :+1: . I will fix it.

Reply:
Ok, I think this solves it https://github.com/airyhq/airy/pull/2241/commits/8e5a0498dcb0e81fd236db527338ddeda8e472f4